t/job_metrics.t

```diff
@@ -95,7 +96,50 @@ describe "HTFeed::JobMetrics" => sub {
 	} while ($kid > 0);
 	ok($jm->get_value($item_metric) == 5);
     };
+    it "reports time w/ second as base unit, at high resolution" => sub {
+        # To do this we sleep 10% of a second (don't want to slow down tests).
+        # Time::HirRes::usleep lets us sleep with microsec precision.
+        # The "tolerance" is somewhat arbitrary, for testing purposes only,
+        # since we're not really testing that Time::HiRes works as promised,
+        # but rather that our assumptions around time are correct.
+        my $t1                     = $jm->time;
+        my $sleep_time             = 0.1;
+        my $sleep_time_in_microsec = $sleep_time * 1000000;
+        my $resolution_tolerance   = 0.01;
 
+        usleep $sleep_time_in_microsec;
+        my $t2      = $jm->time;
+        my $delta_t = $t2 - $t1;
+
+        # Now prove that t1 and t2 are (very close to) 0.25 sec apart.
```

Comment:
I think the logic here is correct, but the actual number doesn't match the `$sleep_time` listed above

Reply:
corrected